docs/resources/lb_route.md

```diff
@@ -48,7 +50,11 @@ The following arguments are supported:
 
 - `backend_id` (Required) - The ID of the backend to which the route is associated.
 - `frontend_id`: (Required) The ID of the frontend to which the route is associated.
-- `match_sni` - (Required) The SNI to match.
+- `match` - (Required) The value to match a redirection.
+    - `sni` - The Server Name Indication TLS extension field from an incoming connection made via an SSL/TLS transport layer.
```

Comment:
Please add a link to the documentation about this

Reply:
There's no specific documentation on this topic for now